src/genophenocorr/preprocessing/_phenotype.py

```diff
@@ -48,8 +52,9 @@ def create_phenotype(self, term_ids: typing.Iterable[typing.Tuple[str, bool]]) -
         for term_id, observed in term_ids:
             term = self._hpo.get_term(term_id)
             if term is None:
-                raise ValueError(f'Term ID {term_id} is not present in HPO v{self._hpo.version}')
-            terms.append((term, observed))
+                self._logger.warning("Term %s cannot be found in HPO version %s. It will be ignored.", term_id, self._hpo.version)
```

Comment:
I think this warning is important but not too useful because it lacks context. I bet the user will be interested in fixing this. However, to do that, he must know about the sample ID. 

However, we do not have the sample ID around here. We may need to restructure this code. I'll make a ticket.

Reply:
Leaving this for now, but agreed a ticket to make this more useful would be helpful.

Reply:
:+1: #113